.eslintrc.json

```diff
@@ -18,7 +18,7 @@
   },
   "plugins": ["react", "@typescript-eslint", "unused-imports"],
   "rules": {
-    "linebreak-style": ["error", "unix"],
+    "linebreak-style": ["error", "windows"],
```

Comment:
Thinking if we should keep this intact to unix based line break. Or use something like the solution mentioned in the accepted answer here: https://stackoverflow.com/questions/39114446/how-can-i-write-a-eslint-rule-for-linebreak-style-changing-depending-on-windo